docs/components/textarea-field/index.md

```diff
@@ -66,6 +66,20 @@ Provide a string to the `@hint` component argument or content to `:hint` named b
 </Form::Fields::Textarea>
 ```
 
+## Character Counter
+
+Optional.
+
+A character counter is exposed via a `secondary` block. It can be combined with any of the other named blocks or label/hint arguments.
+
+```hbs
+<Form::Fields::Textarea class='w-full' @label='Label' @hint='With hint text'>
+  <:secondary as |secondary|>
+    <secondary.CharacterCount @max={{255}} />
+  </:secondary>
+</Form::Fields::Textarea>
+```
```

Comment:
Suggestion: it would be nice to have this in the `changeset` demo...maybe as a follow up.

But then again, we don't have analytics (do we?) so we don't know which docs folks are looking at the most as an example...

Reply:
Yeah! I was going to add this in the docs later.   There's an issue I filed around maybe making components all full-width, otherwise the demo looks kinda funky.  I'll work on this next week hopefully though!